### PR TITLE
New UI on Kotlin

### PR DESCRIPTION
**Pull request by Fatalll**

### CODE REVIEW

File: src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            if (cblEnabledLanguages.isItemSelected(it.name)) {
+                settings.state.enabledLanguages.add(it)
+            } else {
+                settings.state.enabledLanguages.remove(it)
+            }
+        }
+
+        settings.state.nativeLanguage = cmbNativeLanguage.selectedItem as Lang
+        settings.state.enabledSpellcheck = cbEnableGraziSpellcheck.isSelected
+        GraziPlugin.reinit()
```
Comment (TanVD) [inline src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt:45]: Merge master please

File: src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import tanvd.grazi.language.Lang
+import java.awt.*
+import javax.swing.*
+
+
+class GraziSettingsPanel : ConfigurableUi<GraziConfig> {
+
+    private val panelEmptyBorder = BorderFactory.createEmptyBorder(3, 5, 5, 5)
+
+    private val cbEnableGraziSpellcheck = JBCheckBox(GraziBundle.message("grazi.ui.settings.enable.text"))
```
Comment (TanVD) [inline src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt:22]: Hungarian notation, huh?)

File: src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    override fun getComponent(): JComponent {
+        Lang.sortedValues.forEach {
+            cmbNativeLanguage.addItem(it)
+            cblEnabledLanguages.addItem(it.name, it.displayName, false)
+        }
+
+        return JPanel(BorderLayout(0, 0)).apply {
+            add(JBTabbedPane().apply {
```
Comment (TanVD) [inline src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt:64]: All of it looks like a perfect time to write simple DSL :)

Comment (Fatalll) [reply src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt:64]: No, it's not

File: src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt (hunk 2 above)
Comment (Fatalll) [reply src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt:22]: Old habit, sometimes you need to know what kind of element it is...

File: src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt (hunk 1 above)
Comment (Fatalll) [reply src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt:45]: Done

**Pull request merged**